ci-operator/config/openshift/runtime-utils/openshift-runtime-utils-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+promotion:
+  name: "4.3"
+  namespace: ocp
```

Comment:
should this actually be promoted?  I don't think so...

Reply:
@petr-muller @stevekuznetsov do we need promotion in order to get the release branch automation?

@bparees I copied the ci-operator config for library-go, and it is using promotion.

Reply:
as far as i know promotion is just about having the images tagged into the ocp/4.3 imagestream which i would not expect you to be doing for this since your "image" is only being built to run tests.

Reply:
> @bparees I copied the ci-operator config for library-go, and it is using promotion.

after we sort out my comments here we should revisit library-go then, i don't understand its use of either `promotion` nor `test_binary_build_commands` and `test-bin`.

Reply:
@bparees, like @adambkaplan says, the promotion destination is used by the tooling to recognize repos and configs that should follow openshift branching & release patterns, so it is correct to have it.

Reply:
i love side-effects.....